server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaBuilder.java

```diff
@@ -295,6 +298,15 @@ public CentralDogmaBuilder numRepositoryWorkers(int numRepositoryWorkers) {
         return this;
     }
 
+    /**
+     * Sets the maximum allowed age of removed projects and repositories before they are purged.
+     * If unspecified, the default of {@value #DEFAULT_MAX_REMOVED_REPOSITORY_AGE_MILLIS} is used.
+     */
+    public CentralDogmaBuilder maxRemovedRepositoryAgeMillis(long maxRemovedRepositoryAgeMillis) {
```

Comment:
Could you also add a method that accepts a `Duration`?

Reply:
Oops, I missed it!